.claude/commands/setup-ci.md

```
@@ -0,0 +1,689 @@
+```bash
+# ファイル存在確認
+ls -la package.json next.config.* tsconfig.json pnpm-workspace.yaml *.tf 2>/dev/null
+```
+
+### 検出ルール
+
+| 条件                                      | タイプ     |
+| ----------------------------------------- | ---------- |
+| `next.config.*` が存在                    | nextjs     |
+| `pnpm-workspace.yaml` または `lerna.json` | monorepo   |
+| `*.tf` ファイルが存在                     | terraform  |
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)